.github/workflows/testcafe_tests.yml

```diff
@@ -0,0 +1,68 @@
+name: TestCafe tests
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  testcafe:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+    - name: Get sources
+      uses: actions/checkout@v2
+
+    # - name: Restore npm cache
+    #   uses: actions/cache@v2
+    #   with:
+    #     path: ./node_modules
+    #     key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
+    #     restore-keys: |
+    #       ${{ runner.os }}-node
+
+    - name: Run npm install
+      run: |
+        sed -i -- 's/21.2-next/21.1-next/' ./package.json
+        npm install --no-package-lock --no-audit --no-fund
+
+    - name: Clone devextreme repo from PR author fork
+      continue-on-error: true
+      if: github.event_name == 'pull_request'
+      run: git clone -b ${{github.event.pull_request.head.ref}} https://github.com/${{github.event.pull_request.user.login}}/devextreme ../devextreme-repo
+
+    - name: Clone devextreme repo
+      run: test -d ../devextreme-repo || git clone -b 21_2 https://github.com/devexpress/devextreme ../devextreme-repo
```

Comment:
Can we use smth like GITHUB_BASE_REF instead of branch name hardcoding. devextreme and devextreme-demos has the same branch naming